src/htmx.js

```diff
@@ -3834,11 +3834,10 @@ return (function () {
         function insertIndicatorStyles() {
             if (htmx.config.includeIndicatorStyles !== false) {
```

Comment:
I wonder if it would be safe to do this.
```suggestion
            if (htmx.config.includeIndicatorStyles) {
```

Reply:
I've added this change as well.